app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
...besides, the table was called `qtl` to begin with and I didn't really see a reason to change it. I suppose it could be changed to `quantitative_trait_loci` but then again, I've recently been receiving complaints about coming up with overly winding names. :)